webui/react/src/pages/TrialDetails.tsx

```diff
@@ -50,7 +50,7 @@ const TrialDetailsComp: React.FC = () => {
   const navigate = useNavigate();
   const { experimentId, tab, trialId: trialID } = useParams<Params>();
   const [tabKey, setTabKey] = useState<TabType>(tab ?? DEFAULT_TAB_KEY);
-  const [trialId, setTrialId] = useState<number>(parseInt(trialID ?? ''));
+  const [trialId, setTrialId] = useState<number>(Number(trialID));
```

Comment:
changed it to `Number` cuz theoretically its faster and we always expect numeric value from `trialID`